### PR TITLE
LPD-4174 Integration and functional test

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/LPD-18765

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:playwright

**Comment by dsitu** [comment]
ci:stop

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:playwright

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java
Comment (dsitu) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java:1]: @luca-pellizzon I discovered this after I did a rebase, we should probably have someone from the Object team take a look.

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
https://github.com/liferay-commerce/liferay-portal/pull/4839

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java
Comment (linolaoj) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java:1]: Hi @dsitu , @luca-pellizzon this was introduced to fix the error reported on [LPS-195674](https://liferay.atlassian.net/browse/LPS-195674). If we follow the steps from LPS-195674 we will get the error `java.sql.Timestamp cannot be cast to java.lang.Long`.  So we might need to check if the `termValues.get(objectField.getName())` returns an instance of `Timestamp` or `Long`.

Comment (dsitu) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/NotificationTemplateObjectActionExecutorImpl.java:1]: @linolaoj opened up a new PR, look [here](https://github.com/liferay-commerce/liferay-portal/pull/4842/commits/c57bc32746194282dd2a5c9a3edc34803aa073ca). Thanks!

**Pull request closed without merging**